docs/tutorials/custom-vnet.md

```diff
@@ -89,40 +92,28 @@ Depending on the orchestrator you want to deploy, the number of agent pools, the
 
 There are a lot of examples available on the [AKS Engine GitHub](https://github.com/Azure/aks-engine/tree/master/examples) and you can find [one dedicated for virtual network](https://github.com/Azure/aks-engine/blob/master/examples/vnet/README.md).
 
-In this case, we are going to use the following template:
+In this case, we are going to use the following template (this creates a cluster with AzureCNI network plugin):
```

Comment:
s/AzureCNI/Azure CNI/ (Add a space)

Reply:
I would also say "**the** Azure CNI network plugin".

Reply:
addressed in https://github.com/Azure/aks-engine/pull/1282/commits/1b31865c36284c0981ad146e7a9dd7001690bd7a